server/entities/database/jury.js

```diff
@@ -6,23 +6,9 @@ const dataTypes = sequelize.DataTypes;
 class Jury extends Model{}
 
 Jury.init({
-    project_id: {
-        type: dataTypes.INTEGER,
-        allowNull: false,
-        primaryKey: true
-    },
-    user_id: {
-        type: dataTypes.INTEGER,
-        allowNull: false,
-        primaryKey: true
-    },
-    grade: {
-        type: dataTypes.FLOAT,
-    
-    },
-    date_graded: {
-        type: dataTypes.DATE
-    },
+    total_grade: {
+        type: dataTypes.FLOAT
+    }
```

Comment:
Do we need this? Since we can compute it with SQL functions

Reply:
I know we can compute through SQL functions, but I think it is alright to have as a field in order to access it easier on front-end

Reply:
yes but it needs to be set as the average of grades awarded by the jury, excluding the lowest and the greatest values. I think it would be hard to manage it further down the line